input-engine/src/main/java/de/tillhub/inputengine/data/Money.kt

```diff
@@ -0,0 +1,84 @@
+package de.tillhub.inputengine.data
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.math.RoundingMode
+
+@Parcelize
+data class Money(
+    val value: BigDecimal,
```

Comment:
Maybe this constructor should be private.

Reply:
having private constrctuor on data class throwing below warning 
![Uploading Screenshot 2024-03-27 at 16.57.02.png…]()